mne/channels/montage.py

```diff
@@ -395,25 +422,56 @@ def __repr__(self):
               ', '.join(self.point_names[:3]))
         return s
 
-    def plot(self, scale_factor=1.5, show_names=False):
-        """Plot EEG sensor montage.
-
-        Parameters
-        ----------
-        scale_factor : float
-            Determines the size of the points. Defaults to 1.5
-        show_names : bool
-            Whether to show the channel names. Defaults to False
-
-        Returns
-        -------
-        fig : Instance of matplotlib.figure.Figure
-            The figure object.
-        """
-        from ..viz import plot_montage
+    @copy_function_doc_to_method_doc(plot_montage)
+    def plot(self, scale_factor=1.5, show_names=False, show=True):
```

Comment:
Noticed that ``plot_montage`` is not using the ``scale_factor``. Could you check that too? Should be an easy fix.